pyproject.toml

```diff
@@ -38,7 +38,9 @@ dependencies = [
     'deepecho>=0.6.0',
     'rdt>=1.12.0',
     'sdmetrics>=0.14.0',
-    'platformdirs>=4.0'
+    'platformdirs>=4.0',
+    'xlsxwriter>=3.1.0',
+    'openpyxl>=3.1.0'
```

Comment:
After speaking with @npatki I think we should make these optional dependencies. I would also just refer to pandas' dependencies instead of maintaining this ourselves. Something like
```toml
[project.optional-dependencies]
excel_handler = [
    'pandas[excel]'
]
```